docs/emu_mps/advanced/config.md

```diff
@@ -32,10 +36,36 @@ In addition to the above procedure, at each truncation step, no more than `max_b
 
-Time evolution of each qubit pair is done by the [Lanczos algorithm](https://en.wikipedia.org/wiki/Lanczos_algorithm). This algorithm works by iteratively constructing a basis in which the Hamiltonian is close to diagonal. In this basis, the time-evolution of the state can efficiently be approximated by exponentiating a truncated Hamiltonian. The algorithm will never construct more than `max_krylov_dim` basis vectors, limiting the runtime and memory consumption of the time-evolution.
+Time evolution of each qubit pair is done by the [Lanczos algorithm](https://en.wikipedia.org/wiki/Lanczos_algorithm). This algorithm works by iteratively constructing a basis in which the Hamiltonian is close to diagonal. In this basis, the time-evolution of the state can efficiently be approximated by exponentiating a truncated Hamiltonian in the case of TDVP, or minimizing the truncated Hamiltonian in the DMRG case. The algorithm will never construct more than `max_krylov_dim` basis vectors, limiting the runtime and memory consumption of the time-evolution.
 
 Note that the number of iterations the Lanczos algorithm needs to converge to the required tolerance depends on the `dt` parameter also present in the config (see the [api specification](../api.md#mpsconfig)). The default value of `max_krylov_dim` should work for most reasonable values of `dt`, so if you get a recursion error out of the Lanczos algorithm, ensure you understand how the [errors in TDVP](errors.md) depend on `dt`.
 
 ## extra_krylov_tolerance
 
 In addition to the above hard cap on the number of basis vectors, the algorithm will also attempt to estimate the error incurred by computing the matrix exponential using only the current basis vectors. In principle, it is not needed to compute the time-evolution more precisely than `precision` since extra precision will be lost in the truncation. However, in practice it turns out that existing error estimates tend to underestimate the error. `extra_krylov_tolerance` is a fudge factor for how big the desired precision should be compared to `precision`. Its default value is `1e-3`.
+
+## num_gpus_to_use
+
+The `num_gpus_to_use` parameter sets the number of GPUs over which the MPS tensors are distributed during the simulation.
+Setting `num_gpus_to_use = 0` runs the entire computation on the CPU.
+Using multiple GPUs can reduce memory usage per GPU, though the overall runtime remains similar.
+
+**Example:**
+num_gpus_to_use = 2  # use 2 GPUs if available, otherwise fallback to 1 or CPU
+
+## optimize_qubit_ordering
+The `optimize_qubit_ordering` parameter enables the reordering of qubits in the register. This can be useful in cases where the initial qubit ordering (chosen by the user) is not optimal. In such cases, setting `optimize_qubit_ordering = True` re-orders the qubits more efficiently, and that has been shown to improve performance and accuracy. The default value is `False`.
+
+**Note:** enabling this option disables certain features.
```

Comment:
Could you say "Enabling this option is not compatible with certain features, such as reading a user-provided initial state."